src/pymovements/dataset/dataset_files.py

```diff
@@ -560,7 +560,7 @@ def load_precomputed_event_file(
     return PrecomputedEventDataFrame(data=precomputed_event_df)
 
 
-def load_text_stimuli_files(
+def load_stimuli_files(
```

Comment:
I'll create a PR for an `@experimental` property that raises an `ExperimentalWarning` if this function is called